src/store/Items/thunks.ts

```diff
@@ -0,0 +1,13 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+// import { ItemProps } from '@components/Item/item.type';
+
+const BASE_URL = 'http://localhost:5000/items';
```

Comment:
```suggestion
const BASE_URL = `${APP_HOST}/items`;
```
E đưa http://localhost:5000 vào biến env APP_HOST nha